dylint/src/driver_builder.rs

```diff
@@ -168,12 +171,37 @@ fn build(opts: &crate::Dylint, toolchain: &str, driver: &Path) -> Result<()> {
     command.success()?;
 
     copy(
-        package
-            .join("target")
-            .join("debug")
-            .join(format!("dylint_driver-{}", toolchain)),
+        package.join("target").join("debug").join(format!(
+            "dylint_driver-{}{}",
+            toolchain,
+            consts::EXE_SUFFIX
+        )),
         driver,
     )?;
 
+    // To succesfully determine the dylint driver Version on Windows,
+    // it is neccesary to place copies of the toolchain dll's next to the driver.
+    #[cfg(target_os = "windows")]
+    {
+        let path = PathBuf::new()
+            .join(env::var(env::RUSTUP_HOME)?)
```

Comment:
```suggestion
        let rustup_home = var(env::RUSTUP_HOME)?;
        let path = PathBuf::from(rustup_home)
```
When an environment variable cannot be found,`dylint_internal::env::var` tells you which one.